index.js

```diff
@@ -63,7 +63,30 @@ module.exports = {
 
     if (isProductionBuild || isOldEmber) {
       tree = new Funnel(tree, {
-        exclude: [/instance-initializers\/axe-component|violations-helper.js/]
+        exclude: [/instance-initializers\/(axe-component|violations-helper)\.js/]
+      });
+    }
+
+    return tree;
+  },
+
+  /**
+   * Exclude all addon code during build if this is a
+   * production build or if the version of Ember being used is less than 1.13.
+   * @override
+   */
+  treeForAddon: function() {
+    var tree = this._super.treeForAddon.apply(this, arguments);
+    var checker = new VersionChecker(this);
+    var isProductionBuild = process.env.EMBER_ENV === 'production';
+    var isOldEmber = checker.for('ember', 'bower').lt('1.13.0');
+
+    if (isProductionBuild || isOldEmber) {
+      tree = new Funnel(tree, {
```

Comment:
Why isn't this tree returned? I think just building it but not returning it will have no effect...

Reply:
It is being returned on line 93. It's just the way the diff is being displayed, makes it not very apparent.

Reply:
D'oh! Sorry about that!